src/XIVLauncher.Common.Unix/Compatibility/WineSettings.cs

```diff
@@ -19,15 +19,18 @@ public class WineSettings
     public string EsyncOn { get; private set; }
     public string FsyncOn { get; private set; }
 
+    public string DxvkFPSLimit { get; private set; }
+
     public string DebugVars { get; private set; }
     public FileInfo LogFile { get; private set; }
 
     public DirectoryInfo Prefix { get; private set; }
 
-    public WineSettings(WineStartupType? startupType, string customBinPath, string debugVars, FileInfo logFile, DirectoryInfo prefix, bool? esyncOn, bool? fsyncOn)
+    public WineSettings(WineStartupType? startupType, string customBinPath, string debugVars, FileInfo logFile, DirectoryInfo prefix, bool? esyncOn, bool? fsyncOn, string dxvkFPSLimit)
```

Comment:
This is not a wine setting and should probably go into the appropriately named `Dxvk` class